suite-native/accounts/src/components/AccountListItem.tsx

```diff
@@ -24,6 +25,20 @@ export type AccountListItemProps = {
     disabled?: boolean;
 };
 
+const TokenBadge = React.memo(({ accountKey }: { accountKey: AccountKey }) => {
+    const numberOfTokens = useSelector((state: TokensRootState) =>
+        selectNumberOfAccountTokensWithFiatRates(state, accountKey),
+    );
+
+    return (
+        <Badge
+            elevation="1"
+            size="small"
+            label={<Translation id="accountList.numberOfTokens" values={{ numberOfTokens }} />}
```

Comment:
Should this return `+1 tokens` instead of `+1 token` ? for one token?

Reply:
Good point but I don't want to do that in this PR